ui-v2/src/routes/runs/flow-run.$id.tsx

```diff
@@ -1,16 +1,9 @@
-import { DeploymentsEmptyState } from "@/components/deployments/empty-state";
-import { DeploymentsPageHeader } from "@/components/deployments/header";
 import { createFileRoute } from "@tanstack/react-router";
 
 export const Route = createFileRoute("/runs/flow-run/$id")({
 	component: RouteComponent,
 });
 
 function RouteComponent() {
-	return (
-		<div className="flex flex-col gap-4">
-			<DeploymentsPageHeader />
-			<DeploymentsEmptyState />
-		</div>
-	);
```

Comment:
This ended up in the wrong route. It's probably my fault, so I thought I'd try and sneak a fix in here 😅